build.yaml

```diff
@@ -2082,37 +2082,40 @@ steps:
       - hail_base_image
       - build_hail
   - kind: runImage
-    name: make_pip_versioned_docs
+    name: get_pip_versioned_docs
     image:
       valueFrom: hail_pip_installed_image.image
     script: |
-      set -ex
+      set -x
```

Comment:
We can restore the `-e`, right? Failing on an error doesn't apply to the expression inside the `if` command.